labs/src/main/scala/org/scalalabs/basic/lab01/HelloWorldExercise.scala

```diff
@@ -79,7 +82,7 @@ object HelloWorld {
 object HelloWorldClassAndObject {
 	def apply(initialText:String):HelloWorldClassAndObject = {
 		new HelloWorldClassAndObject {
-			val text="FixMe"
+			val text="Hello"
```

Comment:
Similarly to above, I’m pretty sure the intended definition was `val text = initialText`.